### PR TITLE
data: Soumava-221B

### PR DESCRIPTION
**Pull request by Soumava-221B**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7500"></a>

### CODE REVIEW

File: data/Soumava-221B.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+"name": "Soumava Das",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am currently learning javascript",
+  "avatar": "https://github.com/Soumava-221B.png",
```
Comment (Sindhuinti) [inline data/Soumava-221B.json:6]: Profile will be auto-imported from your github no need to add avatar link

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/Soumava-221B.json (hunk 1 above)
Comment (Soumava-221B) [reply data/Soumava-221B.json:6]: Okay, i will remove the avatar link and commit the changes

File: data/Soumava-221B.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+"name": "Soumava Das",
+  "displayStatsPublic": true,
```
Comment (krishstwt) [inline data/Soumava-221B.json:3]: This is a deprecated field :D 
```suggestion
```

File: data/Soumava-221B.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+"name": "Soumava Das",
+  "type": "personal",
```
Comment (SaraJaoude) [inline data/Soumava-221B.json:3]: this is also no longer necessary:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**